# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/RemoteTrackSource.h
# Commit: f4d3d1b63c2c
# Full Hash: f4d3d1b63c2ccf3bbcb5bf30792a9781b8f98a09
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2025-08-08 21:23:04
# Description:
#   Bug 1975995 - Fix remote track getSettings after peer connection closed. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D260391
# ==============================================================================

diff -r 82915b38e93e -r f4d3d1b63c2c dom/media/webrtc/jsapi/RemoteTrackSource.h
--- a/dom/media/webrtc/jsapi/RemoteTrackSource.h	Fri Aug 08 15:25:10 2025 +0300
+++ b/dom/media/webrtc/jsapi/RemoteTrackSource.h	Fri Aug 08 12:34:03 2025 +0000
@@ -61,6 +61,7 @@
 
   RefPtr<SourceMediaTrack> mStream;
   RefPtr<dom::RTCRtpReceiver> mReceiver;
+  Maybe<gfx::IntSize> mReceivingSizeOnEnded;
 };
 
 }  // namespace mozilla